shared/src/commonMain/resources/assets/values-tr-TR/strings.xml

```diff
@@ -1,256 +1,341 @@
 <resources>
-    <string name="language_name">Türkçe</string>
-    <string name="project_url_name">proje bağlantısı</string>
-    <string name="info_using_release_$x">$x sürümünü kullanma</string>
-    <string name="info_using_non_release_commit_$x">Yayınlanmama taahhüdünü kullanma $x</string>
-    <string name="version_message_release_$appver_$apiver_$split">SpMp sürümü build $appver $splitand API version $apiver</string>
-    <string name="version_message_non_release_$commit_$apiver_$split">$commit $ splitand API sürümü $apiver üzerinde SpMp yayınlanmayan derleme</string>
-    <string name="help_message">YAPILACAK</string>
+    <string name="language_name">Türkçe (TR)</string>
+
+    <string name="version_string" translatable="false">0.4.0-beta1</string>
+    <string name="version_code" translatable="false">14</string>
+
+    <string name="app_name" translatable="false">SpMp</string>
+    <string name="app_name_debug" translatable="false">SpMp (debug)</string>
+    <string name="project_url" translatable="false">https://github.com/toasterofbread/spmp</string>
+    <string name="project_license" translatable="false">GPL-3.0</string>
+    <string name="project_author" translatable="false">toasterofbread</string>
+    <string name="report_issue_url" translatable="false">https://github.com/toasterofbread/spmp/issues/new?labels=Bug&amp;template=bug_report.md&amp;title=%5BBug%5D+</string>
+    <string name="server_info_url" translatable="false">https://spmp.toastbits.dev/docs/latest/server/about/</string>
+    <string name="documentation_url" translatable="false">https://spmp.toastbits.dev/docs/latest/about</string>
+    <string name="server_documentation_url" translatable="false">https://spmp.toastbits.dev/docs/latest/server/about</string>
+    <string name="flatpak_documentation_url" translatable="false">https://spmp.toastbits.dev/docs/latest/client/installation-flatpak</string>
+    <string name="donation_url" translatable="false">https://liberapay.com/toasterofbread/</string>
```

Comment:
Strings marked with translatable="false" should not be included (this is stated in [the localisation wiki page](https://github.com/toasterofbread/spmp/wiki/App-localisation)).

Reply:
> Strings marked with translatable="false" should not be included (this is stated in [the localisation wiki page](https://github.com/toasterofbread/spmp/wiki/App-localisation)).

Done.